suse2013/common/labels.xsl

```diff
@@ -17,5 +17,36 @@
 
   <xsl:template match="refsect1/title|refnamediv" mode="label.markup"/>
 
-
+  <xsl:template match="task" mode="label.markup">
+   <xsl:variable name="pchap"
+                 select="ancestor::chapter
+                         |ancestor::appendix
+                         |ancestor::article[ancestor::book]"/>
+   <xsl:variable name="prefix">
+     <xsl:if test="count($pchap) &gt; 0">
+       <xsl:apply-templates select="$pchap" mode="label.markup"/>
+     </xsl:if>
+   </xsl:variable>
+   <xsl:choose>
+    <xsl:when test="@label"><xsl:value-of select="@label"/></xsl:when>
```

Comment:
use two spaces for indentation please. (it seems you're switching between one and two occasionally.)

Reply:
Arhg, I was too inconsistent. Could we do it later? Or what procedure do you prefer to change this?